extension/js/common/platform/catch.ts

```diff
@@ -192,8 +192,8 @@ export class Catch {
       env = 'ex:prod';
     } else if (url.indexOf('gjdhkacdgd') !== -1 || url.indexOf('gggocmadhd') !== -1) {
       env = 'ex:dev';
-    } else if (url.indexOf('gjdhkacdgd') !== -1) { // in case it differs in the future
-      env = 'ex:test';
+      // } else if (url.indexOf('gjdhkacdgd') !== -1) { // in case it differs in the future
```

Comment:
This branch can never execute because of the previous branch.

Reply:
Thanks :+1: